src/frontend/App/hooks/useOppgave.ts

```diff
@@ -92,7 +92,7 @@ export const useOppgave = (oppgave: IOppgave) => {
             .finally(() => settLaster(false));
     };
 
-    const gåTilFagsak = (personIdent: string) => {
+    const plukkOppgaveOgGåTilBehandlingsoversikt = (personIdent: string) => {
```

Comment:
Er dette egentlige `plukkOppgaveOgGåTilJournalføring` ?

Reply:
Den går til behandlingsoversikten for tilbakekreving og klage, så tror nåværende navn er korrekt